src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -20,7 +21,35 @@ def get(self, request, organization):
         if not features.has("organizations:discover-basic", organization, actor=request.user):
             return self.get_v1_results(request, organization)
 
+        top_events = "topEvents" in request.GET
+        limit = None
+
+        if top_events:
+            try:
+                limit = int(request.GET.get("topEvents", 0))
+            except ValueError:
+                return Response({"detail": "topEvents must be an integer"}, status=400)
+            if limit > MAX_TOP_EVENTS:
+                return Response(
+                    {"detail": "Can only get up to {} top events".format(MAX_TOP_EVENTS)},
+                    status=400,
+                )
+            elif limit <= 0:
+                return Response({"detail": "If topEvents needs to be at least 1"}, status=400)
+
         def get_event_stats(query_columns, query, params, rollup, reference_event):
+            if top_events:
+                return discover.top_events_timeseries(
+                    timeseries_columns=query_columns,
+                    selected_columns=request.GET.getlist("field")[:],
+                    user_query=query,
```

Comment:
Why use a different name than the other methods in this module? To the external consumer this is their 'query'.

Reply:
if I use `query` I can't use the `query` function anymore.